repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala

```diff
@@ -150,6 +149,7 @@ class PythonInterpreterSpec extends BaseInterpreterSpec {
       "name 'x' is not defined",
       List(
         "Traceback (most recent call last):\n",
+        "  File \"<stdin>\", line 1, in <module>\n",
```

Comment:
I wonder how useful this is? Is there a proper correlation between the line shown here and anything the user has submitted?

From what I remember there is not a 1:1 relationship between a statement submission and a "line" sent to the interpreter.